pkg/services/sqlstore/sqlstore.go

```diff
@@ -296,7 +296,7 @@ func (ss *SQLStore) buildConnectionString() (string, error) {
 		}
 
 		if ss.Cfg.IsFeatureToggleEnabled("mysqlAnsiQuotes") || ss.Cfg.IsFeatureToggleEnabled("newDBLibrary") {
-			cnnstr += "&sql_mode='ANSI_QUOTES'"
+			cnnstr += "&sql_mode='ANSI_QUOTES'&parseTime=true"
```

Comment:
This would change the behavior of "mysqlAnsiQuotes" even if there isn't the "newDBLibrary" toggle. Should these be kept separate. Also perhaps true of the test conn_str?

Reply:
separated the 2 of them